test_runner/regress/test_protocol_version.py

```diff
@@ -0,0 +1,9 @@
+from fixtures.neon_fixtures import NeonEnv
+
+
+def test_protocol_version(neon_simple_env: NeonEnv):
```

Comment:
Extending this test to restart the compute and do a few reads with the new protocol version would give me some extra peace of mind.

Reply:
Please notice that to execute even simplest query as `select 1` or `show neon.protocol_version` Postgres actually has to load a lot of pages from system catalog.

Reply:
Makes sense, but I think it would still make sense to restart with the new protocol version and execute `show neon.protocol_version`.